E: Unable to locate package libvpx5

I just try this pack but i got this errors ;

```
remote:        Get:57 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 mesa-va-drivers amd64 20.2.6-0ubuntu0.20.04.1 [2,618 kB]
remote:        Get:58 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 mesa-vdpau-drivers amd64 20.2.6-0ubuntu0.20.04.1 [2,744 kB]
remote:        Get:59 http://archive.ubuntu.com/ubuntu focal/universe amd64 i965-va-driver amd64 2.4.0-0ubuntu1 [924 kB]
remote:        Get:60 http://archive.ubuntu.com/ubuntu focal/universe amd64 va-driver-all amd64 2.7.0-2 [4,020 B]
remote:        Get:61 http://archive.ubuntu.com/ubuntu focal/main amd64 vdpau-driver-all amd64 1.3-1ubuntu2 [4,596 B]
remote:        Fetched 44.9 MB in 2s (21.0 MB/s)
remote:        Download complete and in download only mode
remote: -----> Fetching .debs for libvpx5
remote:        Reading package lists...
remote:        Building dependency tree...
remote: E: Unable to locate package libvpx5
remote:  !     Push rejected, failed to compile Playwright app.
remote:
remote:  !     Push failed
remote: Verifying deploy...
remote:
remote: !       Push rejected to anobesaz.
remote:
To https://git.heroku.com/anobesaz.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://git.heroku.com/anobesaz.git'

```

The order of playwright buildpack and nodejs is correct.

I managed to deploy the buildpack. I forked the project (https://github.com/wrone/heroku-playwright-buildpack/blob/master/bin/compile), removed all dependencies and copied only chromium ones from here https://github.com/microsoft/playwright/blob/master/utils/docker/Dockerfile.bionic.

Now I can't execute my script - Error: Cannot find module 'playwright-chromium'.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -5,8 +5,17 @@ on:
   pull_request:
     branches: [ master ]
 jobs:
-  build:
+  test_ubuntu_18:
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@v2
     - run: bash test/run.sh
+      env:
+        STACK: heroku-18 
+  test_ubuntu_20:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - run: bash test/run.sh
+      env:
+        STACK: heroku-20 

--- a/bin/compile
+++ b/bin/compile
@@ -58,11 +58,25 @@ libgdk-pixbuf2.0-0
 libcairo-gobject2
 # needed for CR recordings and FF video playback
 ffmpeg
-libvpx5
 libpulse0
 libusb-1.0-0
 EOF
 	fi
+
+	case "$STACK" in
+	  "heroku-18")
+	    cat << EOF >>$build_tmpdir/Aptfile
+libvpx5
+EOF
+	    ;;
+	  "heroku-20")
+	    cat << EOF >>$build_tmpdir/Aptfile
+libvpx6
+EOF
+	    ;;
+	  *)
+	    error "STACK must be 'heroku-18' or 'heroku-20'"
+	esac
 
 	cat << EOF >>$build_tmpdir/Aptfile
 # (Optional) XVFB if there's a need to run browsers in headful mode

--- a/test/run.sh
+++ b/test/run.sh
@@ -12,6 +12,7 @@ main() {
   compile_dir=$(mktmpdir)
   env_dir=$(mktmpdir)
   cp -a "$(pwd)"/* ${bp_dir}
+  export STACK="heroku-20"
   ${bp_dir}/bin/compile ${compile_dir} ${2:-$(mktmpdir)} $env_dir
 }
 

--- a/test/run.sh
+++ b/test/run.sh
@@ -12,7 +12,6 @@ main() {
   compile_dir=$(mktmpdir)
   env_dir=$(mktmpdir)
   cp -a "$(pwd)"/* ${bp_dir}
-  export STACK="heroku-20"
   ${bp_dir}/bin/compile ${compile_dir} ${2:-$(mktmpdir)} $env_dir
 }
 

